src/main/scala/codacy/plugins/DockerTest.scala

```diff
@@ -15,50 +13,54 @@ object DockerTest {
   private lazy val config = Map(
     "all" -> possibleTests,
     "allWithUdas" -> possibleTestsWithUda
-  ) ++ possibleTestsWithUda.map { case test =>
+  ) ++ possibleTestsWithUda.map { test =>
     test.opt -> Seq(test)
   }
   private lazy val possibleTests = Seq(JsonTests, PluginsTests, PatternTests)
   private lazy val possibleTestsWithUda = SourceTests +: possibleTests
   private lazy val possibleTestNames = config.keySet
 
-  def main(args: Array[String]) {
+  def main(args: Array[String]): Unit = {
     val typeOfTests = args.headOption
-    val dockerImageName = args.drop(1).headOption
+    val dockerImageNameAndVersionOpt = args.drop(1).headOption
     val optArgs = args.drop(2)
 
-    typeOfTests.collect { case typeOfTest if possibleTestNames.contains(typeOfTest) =>
-      dockerImageName.map { dockerName =>
-        val plugin = new DockerPlugin(DockerImageName(dockerName))
-        val testSources = DockerHelpers.testFoldersInDocker(plugin.dockerImageName)
+    typeOfTests.fold(Printer.red(s"[Missing] test type -> [${possibleTestNames.mkString(", ")}]")) {
+      case typeOfTest if possibleTestNames.contains(typeOfTest) =>
+        dockerImageNameAndVersionOpt.fold(Printer.red("[Missing] docker ref -> dockerName:dockerVersion")) {
+          dockerImageNameAndVersion =>
 
-        val result = possibleTestsWithUda
-          .map(test => run(plugin, testSources, test, typeOfTest, dockerName, optArgs))
-          .forall(identity)
+            val testSources = DockerHelpers.testFoldersInDocker(dockerImageNameAndVersion)
 
-        testSources.foreach(dir => FileUtils.deleteQuietly(dir.toFile))
+            val result = possibleTestsWithUda
```

Comment:
Change from "result" to something like "successfulTests" or "isSuccessful" - something that conveys that the result of the expression is a boolean value that encodes a successful result if all tests were executed with success

Reply:
or "allTestsPassed" maybe

Reply:
done, called it `allTestsPassed ` :+1: